src/containers/Articles/Article/CreateArticle.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { EditorState } from 'draft-js';
+import { Redirect } from 'react-router-dom';
+
+import * as actions from '../../../store/actions/index';
+import CreateArticleForm from '../../../components/UI/Article/ArticleForm';
+import { successToast } from '../../../components/UI/Toast/toast';
+
+export class CreateArticle extends Component {
+
+    state = {
+        editorState: EditorState.createEmpty(),
+        articleData: {
+            title: "",
+            description: "",
+            body: ""
+        },
+        error: false,
+        titleTouched: false,
+        descriptionTouched: false,
+        bodyTouched: false,
+        disabled: false
+    }
+
+    onChangeHandler = (editorState) => {
+        this.setState({editorState})
+    }
+
+    bodyChangedHandler=( event, editor ) => {
+        const data = editor.getData();
+        const currentState = {...this.state}
+        const articleData = {...currentState.articleData}
+        articleData.body = data
+        let disableBtn = true
+        if ((this.state.articleData.body === '' && this.state.articleData.body > 5) || articleData.title === "" || articleData.description === "") {
+            disableBtn = false
+        }
+
+        this.setState({
+            articleData: articleData,
+            disabled: disableBtn
+        })
+    }
+
+
+    inputHandler = (e) => {
+        const currentState = {...this.state}
+        const articleData = {...currentState.articleData}
+        let titleTouched = false
+        let descriptionTouched = false
+        let bodyTouched = false
+        let disableBtn = true
+
+        if (e.target.name === 'title') {
+            articleData.title = e.target.value
+            titleTouched = false
+        } else if (e.target.name === 'description') {
+            articleData.description = e.target.value
+            descriptionTouched = false
+        }
+
+        if (articleData.title !== "") {
```

Comment:
The condition (`!==""`) is unnecessary.   `if (articleData.title)`  is enough.
It is the same issue on line 66.